### DISCUSSION TITLE
News Cache [2023-05-15]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#2465] [Announcements]
### 🚢  Ships

- [Instant Rollback is now generally available to revert deployments](https://vercel.com/changelog/instant-rollback-is-now-available-to-revert-deployments)
- [Automatic recursion protection for Vercel Serverless Functions](https://vercel.com/changelog/automatic-recursion-protection-for-vercel-serverless-functions)

### 💡 Inspiration

- [People are using Svelte to build mock OS sites](https://twitter.com/Rich_Harris/status/1657097805733715987?s=20) 😍
- [The SvelteKit codebase behind HuggingChat app is now open sourced on GitHub!](https://twitter.com/julien_c/status/1656671017547485185?s=20) ⭐ 

### 📚 Resources

- [What does Vercel do?](https://vercel.com/blog/what-is-vercel)
- [Intercepting Routes](https://twitter.com/lydiahallie/status/1657073384046329886?s=20)

**Comment by afkvido** [comment]
typo in "The SvelteKit codebase behind HuggingChat app is **not** open sourced on GitHub!"
Quite sure you meant to say "now"

**Reply by amyegan** [reply]
Fixed. Thanks for letting me know!
